src/Rest/Routes/CacheDeleteRoute.php

```diff
@@ -114,7 +114,7 @@ public function routeCallback(WP_REST_Request $request)
 			'code' => 200,
 			'status' => 'success',
 			// translators: %s will be replaced with the cache type.
-			'message' => \sprintf(\esc_html__('%s cache successfully deleted!', 'eightshift-form'), \ucfirst($type)),
+			'message' => \sprintf(\esc_html__('%s cache successfully deleted!', 'eightshift-forms'), \ucfirst($type)),
```

Comment:
```suggestion
			'message' => \sprintf(\esc_html__('%s cache deleted successfully!', 'eightshift-forms'), \ucfirst($type)),
```

Just rember the classic word order: _ze bluetoof devaice hez connectid successfulley_ (https://www.youtube.com/shorts/-SqmbbmzNzo)